app/public/index.html

```diff
@@ -76,6 +76,14 @@
     <script src="js/cola.js"></script>
     <script src="js/cytoscape-cola.js"></script>
     <script src="js/jquery.clipurl.js"></script>
+	<!-- Global site tag (gtag.js) - Google Analytics -->
+	<script async src="https://www.googletagmanager.com/gtag/js?id=G-CN60S2F3N4"></script>
+	<script>
+		window.dataLayer = window.dataLayer || [];
+		function gtag(){dataLayer.push(arguments);}
+		gtag('js', new Date());
+		gtag('config', 'G-CN60S2F3N4');
+	</script>
```

Comment:
Just a small, irrelevant detail: it looks like the indent is different than above.